lua/wikis/lab/MatchGroup/Input/Custom.lua

```diff
@@ -13,11 +13,10 @@ local Variables = require('Module:Variables')
 
 local MatchGroupInputUtil = Lua.import('Module:MatchGroup/Input/Util')
 
-local DEFAULT_BESTOF = 3
-
 local CustomMatchGroupInput = {}
 local MatchFunctions = {
 	DEFAULT_MODE = 'team',
+	getBestOf = MatchGroupInputUtil.getBestOf,
```

Comment:
with this you want the removeUnsetMaps again^^

Reply:
Rikard says no